### DISCUSSION TITLE
Thumbnails not showings in some feeds.

### DISCUSSION THREAD

**Discussion by ghost** [spacecowboy/Feeder#358] [Q&A]
The app fails to show some thumbnails (like Al-Jazeera's for example, or El Salto Diario) while showing other feeds pictures.

**I tried the following:**

- Checking other rss apps (like de _News_ rss app from f-droid) and they do load the thumbnails from this sites.
- Also tried using other device.

I would like to keep using Feeder tho. I'm not very tech savvy.
Is this an issue for everyone or is it only from my side?
Is there a way to make the thumbnails appear?

[This is the link I used to add Al-Jazeera.](https://www.aljazeera.com/xml/rss/all.xml)

**Comment by spacecowboy** [comment]
It's a site issue. Some sites do not include regular images. Instead they do Javascript trickery or similar things.
